news/_posts/2023-10-12-sc_subdomain.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Single cell subdomain re-launch: Unified and feedback-driven"
+contributions:
+  authorship: [nomadscientist, pavanvidem, "José Manuel Domínguez"]
```

Comment:
```suggestion
  authorship: [nomadscientist, pavanvidem]
  infrastructure: [kysrpex]
```

we don't support free-text names, @kysrpex should be added to the contributors.yaml (also this is a really nice opportunity to use the `infrastructure` tag to recognise a non-authorship infrastructure maintenance/admin task!!)

Reply:
oh awesome! it'd be good to know what types of tags can exist there
I didn't tag kysrpex because he wasn't in the contributors file but he must be somewhere in infrastructure somewhere? Or I can add him?